### DISCUSSION TITLE
MHIG unit - EU to ROW region conversion

### DISCUSSION THREAD

**Discussion by krezo888** [Mr-MIBonk/M.I.B._More-Incredible-Bash#465] [General]
I have a Japanese import mk7 golf with MHIG unit, I'm trying to change regions and install ROW maps specifically Aus/NZ.

I have successfully converted my unit from **MHIG_JP_VW_P0240** to **MHIG_EU_VW_K1550**, applied the all in one patch, changed nav region setting to Australia but I still couldn't install the Aus/NZ maps. I could only apply fec code for EU maps 09100008, fec codes for Aus/NZ couldn't be added, I guess due to EU region firmware.

Is it possible to convert the unit from **MHIG_EU_VW_K1550** to **MHIG_RW_VW_S1534** ? Then wait for a patch to come out for **MHIG_RW_VW_S1534** where I could apply fec 091600EE for Aus/NZ maps.

Cheers,

**Comment by aledepaoligampel** [comment]
Hi! How did you install the RoW firmware? Didn't you have problems with the train?

**Reply by krezo888** [reply]
I couldn't install RoW firmware as train is set to EU after conversion. I managed to install RoW maps after changing the region to RoW in the variant info setting, but the nav still didn't work which I guess is due to train conflict.

We need a conversion to RoW option to flash the eeprom and also a patch for the RoW firmware as the only solution.

**Reply by pixeleyesltd** [reply]
Did you ever figure this out? I converted from JP firmware to EU, however I cannot get NZ maps installed.

**Reply by pixeleyesltd** [reply]
I managed to get NZ / New Zealand maps on a JP / Japan unit in case you are still needing the details. This is applicable for firmware version MHI2_ER_VWG11_K3342_1_AIO_MU1427_20230318, maybe others. My problem: after EU conversion and firmware update using the AIO firmware method, ROW maps would not load. The M.I.B. add FEC step did not function for me, perhaps due to the AIO procedure. This prevented ROW maps from being installed, but there were other steps involved too as the regions were incorrect. See below steps for how I resolved it.

- Generated FEC SWAP code via https://vwcoding.ru/en/utils/fec/
  - FEC "093d0008" which is ROW Maps indefinitely. See FEC decoder for details: https://mib-helper.com/fec-decoder/?fec=093d0008
  - I used browser web inspector to change one of the HTML checkbox values to the FEC above, which when checked and combined with your VIN and VCRN generated a valid SWAP code for me.
- Loaded SWAP code via OBDEleven adaption channel, and activated with basic setting: https://www.youtube.com/watch?v=rgAVFoQQ6v4. It then shows up in the Installed FEC list.
- Using M.I.B.: 
  - Cleaned the Nav DB database
  - Set Variant region to RoW
  - Set navigation region and SDS region to Australia
  - Reboot
- Load the latest 2024/2025 RoW maps onto appropriately sized FAT32 formatted SD card, downloaded from VW website. If using macOS, I deleted all ._ files in the SD card folder structure after the copy.
- Initiate software update via long press menu > SWUpdate. All NavDB, eggnog etc in the list should say Y, and no errors should occur. The update was successful, and on reboot navigation worked without issue.

Hope that helps someone! NZ maps do work on EU firmware with some coaxing.

**Reply by another-kiwi** [reply]
Thanks Pixeleyes, most helpful instruction particularly the browser inspector trick!

Note for anyone wanting to update an Audi with MIB2 High unit the FEC for RoW is 023d00ee
